scss/_buttons.scss

```diff
@@ -204,7 +204,7 @@
   vertical-align: baseline;
 }
 
-/// @deprecated Compose existing classes instead. To be removed in v4
+/* @deprecated Compose existing classes instead. To be removed in v4 */
```

Comment:
changed comment format as this was causing errors that I could not resolve with rules.